libs/blocks/marquee/marquee.css

```diff
@@ -606,6 +583,17 @@
     max-width: 600px;
   }
 
+  .marquee.small .foreground,
+  .marquee.split .foreground.container {
+    padding: 0;
+  }
+  
+  .marquee.quiet .foreground,
+  .marquee.inline .foreground,
+  .marquee.split .foreground {
```

Comment:
Same here, if `:is` is the new trend 🙂

Reply:
It has become my personal side-mission to make it the new tred 😆